x/wasm/keeper/querier_test.go

```diff
@@ -798,31 +798,51 @@ func TestQueryPinnedCodes(t *testing.T) {
 }
 
 func TestQueryParams(t *testing.T) {
-	ctx, keepers := CreateTestInput(t, false, AvailableCapabilities)
-	keeper := keepers.WasmKeeper
-
-	q := Querier(keeper)
+	specs := map[string]struct {
+		srcPermission types.AccessType
+		expInstConf   types.AccessConfig
```

Comment:
I think these two should be renamed, because they are both src and expected.

Reply:
Thanks. I fixed it.